modules/kubecolor/hm.nix

```diff
@@ -0,0 +1,76 @@
+{ config, lib, ... }:
+
+with config.lib.stylix.colors.withHashtag;
```

Comment:
> LGTM.

Actually, consider removing this "global" `with` to align with https://github.com/danth/stylix/pull/425.

Reply:
Done